cryptoshred-spring-boot-autoconfigure/README.md

```diff
@@ -0,0 +1,17 @@
+Module *cryptoshred-spring-boot-autoconfigure*
+==============================================
+
+This maven module provides simple integration of the *cryptoshred* library into Spring Boot.
+
+
+Configuration
+-------------
+
+The following configuration properties exists:
+
+| **Property name**                | **Description** | **Required** | **Default Value** |
+|----------------------------------|-----------------|---------------|----|
+| `cryptoshred.cloud.aws.dynamo.tablename`| name of AWS DynamoDB table to store the subjectId/ key pairs. Only required when module *cryptoshred-cloud-aws* is also used. | yes | - |
```

Comment:
should it be moved to the respective module's readme, then?

Reply:
thought about it. A (Spring) user of the library will always include the "autoconfigure" module so I see this module as a central starting point where all information should be at hand. Also, we're talking about one property where I found "excluding and just referring" too little value. 
To complete the docs, I added some words about this property in the AWS module instead. I know, the documentation it is now in 2 places (here and the aws module) but since just one property is concerned I found this acceptable.